cmd/mailchain/internal/settings/testdata/TestRoot_ToYaml/comment-defaults.comment-defaults.golden.yaml

```diff
@@ -122,6 +122,12 @@
 #     address: "https://relay.mailchain.xyz/json-rpc/ethereum/rinkeby"
 #   ethereum-rpc2-ropsten:
 #     address: "https://relay.mailchain.xyz/json-rpc/ethereum/ropsten"
+#   substrate-rpc-edgeware-testnet:
+#     address: "ws://testnet3.edgewa.re:9944"
+#   substrate-rpc-kusama-testnet:
+#     address: ""
+#   substrate-rpc-polkadot-testnet:
+#     address: ""
```

Comment:
Mailchain requires the contract module for substrate. At this moment `substrate-rpc-edgeware-testnet` is the only network that supports it. `substrate-rpc-kusama-testnet` and `substrate-rpc-polkadot-testnet` need to be removed.